assets/user-profile/components/UserProfileMenu.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { gettext } from 'utils';
+
+
+function UserProfileMenu({isProfile, isFollowedTopics, isDownloadHistory, onClick}) {
```

Comment:
not sure if we should create single app for all these, any benefits?

Reply:
I can think of few marginal benefits like some re-use of actions and reducers, no need for server round trip for every menu click, Maybe I can advocate for functionality for `user profile` feels like one app. But I can't create a strong case for single app to be honest.

Reply:
lets keep it for now, there is still some slight chance we would need to make it a modal, and then it would be easier